modules/tax/src/Entity/TaxType.php

```diff
@@ -34,7 +34,6 @@
  *   config_export = {
  *     "id",
  *     "name",
- *     "genericLabel",
```

Comment:
I'm not sure about this one, but it gave me some trouble. I removed it as it is a generated thing.